vocabularies/gsq-roles.ttl

```diff
@@ -1,121 +1,131 @@
+    skos:topConceptOf <http://linked.data.gov.au/def/gsq-roles> .
+
+iso11179:Submitter a skos:Concept ;
+    skos:altLabel "Lodger"@en ;
+    skos:definition "A Submitter should be an organizational unit approved by a process defined by the Registration Authority. A Submitter is authorized to identify and report Administered Items suitable for registration. The Submitter can be viewed as a contact for the Submitting Organization."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/gsq-roles>;
+    skos:prefLabel "Submitter"@en ;
+    skos:topConceptOf <http://linked.data.gov.au/def/gsq-roles>;
+    skos:historyNote "In QDEX Reports, this role has been known as 'Lodger' but 'Submitter' is now the preferred term for it."@en .
+
+isorole:author a skos:Concept ;
+    skos:definition "Agent who authored the resource"@en ;
+    skos:inScheme <http://linked.data.gov.au/def/gsq-roles>;
+    skos:historyNote "Agent is known as Party in the original definition from ISO19115"@en ;
+    skos:prefLabel "Author"@en ;
+    skos:topConceptOf <http://linked.data.gov.au/def/gsq-roles>.
+
+isorole:collaborator a skos:Concept ;
+    skos:definition "Party who assists with the generation of the resource other than the principal investigator"@en ;
+    skos:inScheme <http://linked.data.gov.au/def/gsq-roles> ;
+    skos:altLabel "Contributor"@en ;
+    skos:prefLabel "Collaborator"@en ;
+    skos:topConceptOf <http://linked.data.gov.au/def/gsq-roles> .
+
+isorole:editor a skos:Concept ;
+    skos:definition "Party who reviewed or modified the resource to improve the content"@en ;
+    skos:inScheme <http://linked.data.gov.au/def/gsq-roles>;
+    skos:historyNote "Agent is known as Party in the original definition from ISO19115"@en ;
+    skos:altLabel "Reviewer"@en ;
+    skos:prefLabel "Editor"@en ;
+    skos:topConceptOf <http://linked.data.gov.au/def/gsq-roles>.
+
+isorole:originator a skos:Concept ;
+    skos:definition "Party who created the resource"@en ;
+    skos:inScheme <http://linked.data.gov.au/def/gsq-roles> ;
+    skos:altLabel "Sampler"@en ;
+    skos:prefLabel "Originator"@en ;
+    skos:topConceptOf <http://linked.data.gov.au/def/gsq-roles>.
+
+isorole:owner a skos:Concept ;
+    skos:altLabel "tenure holder"@en ,
+        "authority holder"@en ,
+        "permit holder"@en ;
+    skos:definition "Agent that owns the resource"@en ;
+    skos:inScheme <http://linked.data.gov.au/def/gsq-roles>;
+    skos:historyNote "Agent is known as Party in the original definition from ISO19115. The altLabels are drawn from P&G (general provisions) regulation 2017, section 37(4)"@en ;
+    skos:prefLabel "Owner"@en ;
+    skos:topConceptOf <http://linked.data.gov.au/def/gsq-roles>.
+
+isorole:principalInvestigator a skos:Concept ;
+    skos:definition "Key party responsible for gathering information and conducting research"@en ;
+    skos:inScheme <http://linked.data.gov.au/def/gsq-roles> ;
+    skos:altLabel "Observer"@en ;

```

Comment:
P.I.s and observers are not the same thing. Not even close. Even the definition says nothing about a party who observes. 
The altLabel facilitates a suggestion to use PI as the observation agent, which is wrong. the act of observing is independent of research and it needs to be pulled out as its own concept.